.travis.yml

```diff
@@ -1,16 +1,15 @@
-language: python
+sudo: required
```

Comment:
While attempting to match the Python and PostgreSQL versions of the app in the CI test environment I realized that the default Travis environment [didn't support PostgreSQL 9.5.x](https://docs.travis-ci.com/user/database-setup/#Using-a-different-PostgreSQL-Version).  Looking into it further yielded their [Trusty Environment])https://docs.travis-ci.com/user/trusty-ci-environment/), which does support it but requires sudo apparently.

I was a bit torn on this given that this is also a beta environment according to their documentation, so if you're aware of being able to use 9.5.x differently in Travis I'd be very curious to know (if you look at the commit history you'll see I tried a few different things to get it to run correctly).  Alternatively, I'm open to other suggestions here as well if you (or anyone else!) feel there's a better way to achieve parity between the project setup and Travis.